gamenews

```diff
@@ -1,9 +1,4 @@
-2023-03-07 18-00-00:
-	Title: Kautious Kup #6
-	Author: Kaution
-	Content: The first tournament on the new release is upon us: on March 18 at 17:00 UTC we will have our 6th edition of the Kautious Kup for Red Alert. If you would like to sign up or find more information, head over to the official Discord server and check out \#competitions-events.
-	
-2023-02-25 18-00-00:
-	Title: Release 20230225
+2023-07-31 07-00-00:
+	Title: Playtest 20230731
 	Author: PunkPun
-	Content: Today we have a special occasion on our hands. We have been waiting for this moment for almost two years: A new stable OpenRA Release is out now, containing several major new features and many small-scale polish improvements. Head to www.openra.net for the full news post and to download the new release.
+	Content: As summer is in full swing, we are kicking off the playtest series for the upcoming OpenRA release!\n\nNotable changes include:\n • New Red Alert and Tiberium Dawn missions\n • Return of the player color brightness remapping\n • Red Alert balance changes\n • And many more small fixes and features\n\nVisit www.openra.net to grab your download and dive right into the action. Make sure to share your feedback with us on forum, Discord, or GitHub.Happy Playtesting!
```

Comment:
```suggestion
	Content: As summer is in full swing, we are kicking off the playtest series for the upcoming OpenRA release!\n\nNotable changes include:\n • New Red Alert and Tiberium Dawn missions\n • Return of the player color brightness remapping\n • Red Alert balance changes\n • And many more small fixes and features\n\nVisit www.openra.net to grab your download and dive right into the action. Make sure to share your feedback with us on forum, Discord, or GitHub. Happy Playtesting!
```
Also the issue with the list is still present. I suggest moving `And many more small fixes and features` outside of it.

Reply:
oh mb, regressed it in the last push

Reply:
What list?

Reply:
The `Notable changes include` list.